feature/experimental/isis/otg_tests/internal/session/session.go

```diff
@@ -201,9 +220,6 @@ func (s *TestSession) WithISIS() *TestSession {
 // IS-IS configuration of ts.ATETop
 func (s *TestSession) ConfigISIS(ocFn func(*oc.NetworkInstance_Protocol_Isis), ateFn func(*ixnet.ISIS)) {
```

Comment:
I still see the ateFn argument here used in OTG tests, is the intent to remove it later? (If so, why is it no longer needed here?)

Reply:
I removed it for now. That is actually not used in the RT-2.2 ATE test

Reply:
That works. (We'll need to add something back in for base_adjacencies_test, but that will have a different type anyways.)